contentctl/actions/test.py

```diff
@@ -115,10 +105,28 @@ def execute(self, input_dto: TestInputDto) -> bool:
                 f"\tTotal Detections             : {summary.get('total_detections','ERROR')}"
             )
             print(
-                f"\tPassed Detections            : {summary.get('total_pass','ERROR')}"
+                f"\tTotal Tested Detections      : {summary.get('total_tested_detections','ERROR')}"
+            )
+            print(
+                f"\t  Passed Detections          : {summary.get('total_pass','ERROR')}"
+            )
+            print(
+                f"\t  Failed Detections          : {summary.get('total_fail','ERROR')}"
+            )
+            print(
+                f"\tSkipped Detections           : {summary.get('total_skipped','ERROR')}"
```

Comment:
These should be unindented; there could be overlap between the categories of manual and deprecated/experimental